app/views/Home/ProjectItem/index.tsx

```diff
@@ -349,7 +349,7 @@ function ProjectItem(props: RecentProjectItemProps) {
                                     isAnimationActive={false}
                                 />
                                 <Area
-                                    dataKey="count"
+                                    dataKey="activity"
```

Comment:
We should also be able to actually define the label.
We may need translated text or "Activity".
The current approach is very limited.